controllers/login.controller.js

```diff
@@ -0,0 +1,18 @@
+const { validationResult } = require("express-validator");
+
+function LoginController() {
+  this.store = function store(req, res, next) {
+    const errors = validationResult(req);
+
+    if (!errors.isEmpty())
```

Comment:
What if there are errors other than `!errors.isEmpty`? Will they be caught by error handlers higher up the flow of control chain?